services/plot-navigator/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: plot-navigator
+version: 1.0.0
+dependencies:
+- name: plot-navigator
+  version: ">=1.0.0"
```

Comment:
This looks a little odd to me. I'd think you'd want to pin a specific version when deploying in phalanx, but maybe this is fine?